kingpin/actors/aws/ecs.py

```diff
@@ -242,6 +242,57 @@ def _register_task(self, task_definition):
             task_definition_name))
         raise gen.Return((family, revision, task_definition_name))
 
+    @gen.coroutine
+    @dry('Would deregister task definition {0}')
+    def _deregister_task_definition(self, task_definition_name):
+        """Deregisters a task definition.
+
+        Args:
+            task_definition_name: Task Definition name or arn to deregister.
+        """
+        self.log.info(
+            'Deregistering task definition {}'.format(task_definition_name))
+        yield self.thread(
+            self.ecs_conn.deregister_task_definition,
+            task_definition=task_definition_name)
+
+    @gen.coroutine
+    @dry('Would list task definitions')
+    def _list_task_definitions(self, status='ALL', family_prefix=None):
+        """List task definitions.
+
+        Optionally uses a prefix to filter family names.
+        Optionally returns active task definitions.
+
+        Args:
+            status: Type of status to filter with.
+                Valid types are 'ACTIVE', 'INACTIVE', and 'ALL'.
+                Default 'ALL'.
+            family_prefix: Prefix to filter task definition families with.
+                Default None.
+
+        Returns:
+            List of task definitions arns matching specified restrictions.
+        """
+        self.log.info(
+            'Listing task definitions '
+            'with status {} and family prefix {}'.format(
+                status, family_prefix))
+        task_definitions = []
+        next_token = None
+        while True:
```

Comment:
Looking at the API... I don't see a way.
http://boto3.readthedocs.io/en/latest/reference/services/ecs.html#ECS.Client.list_task_definitions